api/src/main/java/io/kaoto/backend/api/resource/v2/IntegrationsResource.java

```diff
@@ -0,0 +1,152 @@
+package io.kaoto.backend.api.resource.v2;
+
+import io.kaoto.backend.api.resource.v1.model.Integration;
+import io.kaoto.backend.api.service.deployment.DeploymentService;
+import io.kaoto.backend.api.service.step.parser.StepParserService;
+import io.kaoto.backend.model.step.Step;
+import io.quarkus.cache.CacheResult;
+import org.eclipse.microprofile.openapi.annotations.Operation;
+import org.eclipse.microprofile.openapi.annotations.parameters.Parameter;
+import org.eclipse.microprofile.openapi.annotations.parameters.RequestBody;
+import org.jboss.logging.Logger;
+import org.jboss.resteasy.reactive.server.ServerExceptionMapper;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * 🐱class IntegrationsResource
+ * 🐱relationship compositionOf DeploymentService, 0..1
+ *
+ * This endpoint will return the yaml needed to deploy
+ * the related integration and the
+ * endpoints to interact with deployments.
+ */
+@Path("/v2/integrations")
+@ApplicationScoped
+public class IntegrationsResource {
+
+    private final Logger LOG = Logger.getLogger(IntegrationsResource.class);
+
+    @Inject
+    public void setDeploymentService(
+            final DeploymentService deploymentService) {
+        this.deploymentService = deploymentService;
+    }
+    private DeploymentService deploymentService;
+
+    @Inject
+    public void setStepParserServices(final Instance<StepParserService<Step>> stepParserServices) {
+        this.stepParserServices = stepParserServices;
+    }
+    private Instance<StepParserService<Step>> stepParserServices;
+
+    /*
+     * 🐱method CRDs: Map
+     * 🐱param dsl: String
+     * 🐱param integration: List<Integration>
+     *
+     * Idempotent operation that given an array of integrations, returns the corresponding CRDs.
+     */
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces("text/yaml")
+    @Path("/")
+    @CacheResult(cacheName = "api")
```

Comment:
Caching (see above comment)

Reply:
How would this cache behave? Is [this the correct "doc" for this annotation](https://es.quarkus.io/guides/cache)?

Reply:
Yes, exactly that one.